doc/website/blog/serverless-graphql-using-aws.md

```diff
@@ -0,0 +1,556 @@
+---
+title: Serverless GraphQL using with AWS and ent
+author: Bodo Kaiser
+authorURL: "https://github.com/bodokaiser"
+authorImageURL: "https://avatars.githubusercontent.com/u/1780466?v=4"
+---
+
+[GraphQL][1] is a query language for HTTP APIs, providing a statically-typed interface to conveniently represent today's complex data hierarchies.
+One way to use GraphQL is to import a library implementing a GraphQL server to which one registers custom resolvers implementing the database interface.
+An alternative way is to use a GraphQL cloud service to implement the GraphQL server and register serverless cloud functions as resolvers.
+Among the many benefits of cloud services, one of the biggest practical advantages is the resolvers' independence and composability.
+For example, we can write one resolver to a relational database and another to a search database.
+
+We consider such a setup using [Amazon Web Services (AWS)][2] in the following. In particular, we use [AWS AppSync][3] as the GraphQL service and [AWS Lambda][4] to run a relational database resolver, which we implement using [Go][5] with [Ent][6] as the entity framework.
+Compared to Nodejs, the most popular runtime for AWS Lambda, Go offers faster start times, higher performance, and, in my opinion, an improved developer experience.
+On the other hand, Ent presents an innovative approach towards type-safe access to relational databases, which in my opinion, is unmatched in the Go ecosystem.
+In conclusion, running Ent with AWS Lambda as AWS AppSync resolvers is an extremely powerful setup to face today's demanding API requirements.
+
+In the next sections, we set up GraphQL in AWS AppSync and the AWS Lambda function running Ent.
+Subsequently, we propose a Go implementation integrating Ent and the AWS Lambda event handler, followed by performing a quick test of the Ent function.
+Finally, we register it as a data source to our AWS AppSync API and configure the resolvers, which define the mapping from GraphQL requests to AWS Lambda events.
+Be aware that this tutorial requires an AWS account and **the URL to a public-accessible Postgres database**, which may incur costs.
+
+### Setting up AWS AppSync schema
+
+To set up the GraphQL schema in AWS AppSync, sign in to your AWS account and select the AppSync service through the navbar.
+The landing page of the AppSync service should render you a "Create API" button, which you may click to arrive at the "Getting Started" page as depicted in the screenshot below.
```

Comment:
Do you mean **bold** text in addition to or as a replacement of the quotation marks?

Reply:
As a replacement to all the currently used quotation marks.

Reply:
Hmm, I am not 100% convinced yet.

Without spending too much thought, I use quotation marks whenever

1. I do an inline quote
2. I "intentionally" misuse a word

At the same time, I use **bold text** whenever I want to strongly draw attention to a word or sentence. Overusing the **bold** formatting could mitigate the intention-drawing effect.

However, I fully agree that this is a very opinionated matter and I tend to defend the quotation marks because it's the option least effort :)

Reply:
It is indeed a matter of personal preference. And since it is your great post I am totally fine with it 😊